src/styles/index.scss

```diff
@@ -1,3 +1,120 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,100&display=swap';
```

Comment:
Avoid using protocol-specific URLs ('http:' or 'https:'). Use protocol-relative URLs instead to prevent issues with mixed content if your site is served over HTTPS.